### DISCUSSION TITLE
Wrong dante in View Log, so no detection.

### DISCUSSION THREAD

**Discussion by NIDcornifleur** [mcguirepr89/BirdNET-Pi#679] [General]
I noticed this morning that I haven't had a detection since yesterday despite a lot of bird activity in my yard.
I go to the View Log and the date displayed is yesterday. I went to look at a few posts here but couldn't find a solution.  Reboot doesn't help. This system, mounted on a CM4, has been running for over 5 months with no problems.  Date here: 18 december
Thanks for your help.

<a href="https://servimg.com/view/19470218/257" target="_blank" ><img  src="https://i54.servimg.com/u/f54/19/47/02/18/screen12.jpg" border="0" alt="Image hébergée par servimg.com" /></a>

**Comment by Scraggarax** [comment]
[This is similar to issue #653](https://github.com/mcguirepr89/BirdNET-Pi/issues/653)

**Reply by NIDcornifleur** [reply]
Not sure if it was going to work, I went to the /pi/BirdSongs folder and it displayed yesterday in two folders as well as today. I deleted the two folders for December 17 and rebooted. Bingo, it works. But the day of the 17th disappeared. There is probably a less invasive solution but I didn't see it. If there was one, I didn't notice. So before you see this as a solution to your similar problem, make sure you are comfortable with losing data. 

<a href="https://servimg.com/view/19470218/258" target="_blank" ><img  src="https://i54.servimg.com/u/f54/19/47/02/18/screen13.jpg" border="0" alt="Image hébergée par servimg.com" /></a>

**Reply by matthew73210** [reply]
Hello,
Next time is stops check how much free space you have. If the drive is full it will stop processing and won't start again. Many such issues have been flagged but the all got taken down a couple weeks ago because they are too complicated to sort at the moment. Notably : #623 #653 etc...
Usally what happens is the processing stops and birdsong filles up with unanalysed media. After purging the folder and a reboot it all starts again.
Cheers

**Comment by NIDcornifleur** [comment]
Thanks for the suggestion. I read the other posts and checked the free space.  47%
